src/dashboard/server/api/validator/config.schema.json

```diff
@@ -66,6 +66,10 @@
               "title": "Whether it supports allowed IP",
               "description": "Set to true if it could be listed in allowed ip page",
               "type": "boolean"
+            },
+            "notifications": {
+              "title": "Notifications for the cluster",
+              "type": "array"
```

Comment:
Shall we specify the type of the array's elements?

Reply:
Good point, element should be string type, but how to implement, sorry for no idea.

Reply:
> Shall we specify the type of the array's elements?

Good point, element should be string type, but how to implement, sorry for no idea.

Reply:
> > Shall we specify the type of the array's elements?
> 
> Good point, element should be string type, but how to implement, sorry for no idea.

https://json-schema.org/understanding-json-schema/reference/array.html#list-validation

Reply:
> > > Shall we specify the type of the array's elements?
> > 
> > 
> > Good point, element should be string type, but how to implement, sorry for no idea.
> 
> https://json-schema.org/understanding-json-schema/reference/array.html#list-validation

thanks for help